source/getting-started/first-app.md

```diff
@@ -7,28 +6,29 @@ ember new my-new-app
 
 This will create a new `my-new-app` directory with your new Ember app inside.
 
-Once the generation process finishes, launch the Ember CLI development server:
+Once the process finishes, launch the Ember development server:
```

Comment:
It is "Ember CLI". Confirmed it with stef before writing this, open to other ideas but seems ok to me.